electron/install.js

```diff
@@ -10,8 +9,14 @@ const axios = require('axios');
 
 const Docker = require('dockerode');
 const { spawnSync } = require('child_process');
-
-const Version = '0.1.0rc41';
+const { BrewScript } = require("./scripts")
+
+/**
+ * current version of the pearl release
+ * - use "" (nothing as a suffix) for latest release candidate, for example "0.1.0rc26"
+ * - use "alpha" for alpha release, for example "0.1.0rc26-alpha"
+ */
+const OlasMiddlewareVersion = '0.1.0rc45';
```

Comment:
can bump for next tag so we don't need to raise another pr
```suggestion
const OlasMiddlewareVersion = '0.1.0rc46';
```